### PR TITLE
Swagger file update for multi-cloud

### PR DESCRIPTION
**Pull request by PravinRanjan10**
This PR is a swagger update for a multi-cloud project
It has improved/correction  of some multi-cloud API requests and responses

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind new feature
> /kind bug fix
> /kind cleanup
> /kind revert change
> /kind design
> /kind documentation
> /kind enhancement

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Test Report Added?**:
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
> /kind TESTED
> /kind NOT-TESTED

**Test Report**:
1. Create Backend

Aws

2. List All Backend

List All Backend

3. List  Backend

List Backend

4. List All Type

List all Type

**Special notes for your reviewer**:

### CODE REVIEW

File: openapi-spec/swagger.yaml [hunk 1]
```diff
@@ -1537,6 +1598,171 @@ components:
...
+          example: fusionstorage-object
+        description:
+          description: The backend type description.
+          type: string
+          example: Huawei Fusionstorage Object Storage
+    IBM_Type:
+      type: object
+      properties:
+        name:
+          description: The backend type name.
```
Comment (kumarashit) [inline openapi-spec/swagger.yaml:1660]:
```suggestion
          description: The backend type.
```

File: openapi-spec/swagger.yaml [hunk 2]
```diff
@@ -1537,6 +1598,171 @@ components:
...
+          example: ibm-cos
+        description:
+          description: The backend type description.
+          type: string
+          example: IBM Cloud Object Storage
+    YIG_Type:
+      type: object
+      properties:
+        name:
+          description: The backend type name.
```
Comment (kumarashit) [inline openapi-spec/swagger.yaml:1671]: Similar here

File: openapi-spec/swagger.yaml (hunk 1 above)
Comment (PravinRanjan10) [reply openapi-spec/swagger.yaml]: Done

**Review by joseph-v — APPROVED** [review]
LGTM

**Review by kumarashit — APPROVED** [review]
LGTM

**Review by anvithks — APPROVED** [review]
LGTM

**Pull request merged**